crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -890,3 +890,149 @@ xxx....vlv....xxx
      .......
      ...T...
 ENDMAP
+
+NAME:   dreamdust_ecumenical_altar_hidden_monk_grotto
+TAGS:   no_monster_gen no_item_gen no_rotate
+WEIGHT: 5
+KMONS:  p = plant / fungus
+KMONS:  b = butterfly
+KITEM:  ! = never_decay human skeleton, robe mundane, quarterstaff mundane
+SUBST:  ? = pbW
+: ecumenical_altar_setup(_G)
+MAP
+xxxxxxxxxxxxxxx
+xxxxxxt_txxxxxx
+xxxxp.!...pxxxx
+xxxp..?W?..pxxx
+xxxx..?W?..xxxx
+xxxxxp...pxxxxx
+xxxxxxx.xxxxxxx
+xxxxxp.xxpxxxxx
+xxxxxxp...xxxxx
+xx..pxxxxp.xxxx
+x.xx....xxx.pxx
+xp.xxxxx...pxxx
+xx.x...xxxxxxxx
+xxx.xpx.x..xxxx
+xxxxxxxp.xx.pxx
+xxxxxxxxxxxx@xx
+ENDMAP
+
+NAME:  dreamdust_ecumenical_altar_angels_and_demons
+TAGS:  transparent
+KFEAT: p = granite_statue
+KFEAT: q = granite_statue
+TILE:  G = dngn_crumbled_column
+TILE:  p = dngn_statue_angel
+TILE:  q = dngn_statue_demonic_bust
+FTILE: -_Gpq = floor_hall
```

Comment:
I can do this when merging, but it's important to set the feature descriptions so that console players get these descriptions.